operator-common/src/main/java/io/strimzi/operator/common/featuregates/FeatureGates.java

```diff
@@ -18,12 +18,11 @@
 public class FeatureGates {
     /* test */ static final FeatureGates NONE = new FeatureGates("");
 
-    // As we currently don't have any real feature gate, this dummy feature gate is used to make it clear how to add a
-    // real feature gates in the future and allows us to test the feature gates.
-    private static final String DUMMY_FEATURE_GATE = "DummyFeatureGate";
```

Comment:
could we leave the above code commented and as place holder maybe changing a little bit the comment in order to make clear what to do when there are no feature gates anymore?
In general I would leave the dummy feature gate code around but commented out.

Reply:
I can do that, I removed it because I thought that we don't want to keep commented code like that

Reply:
Let's see what others think. It's my opinion, I am not pushing you to make the change.

Reply:
Sure thing, I have the changes ready, so other @strimzi/maintainers can check and leave their opinion here.